gemstone/tests/ios/GemTest/GemTest/Provider.swift

```diff
@@ -20,7 +20,7 @@ public actor NativeProvider {
 }
 
 extension NativeProvider: AlienProvider {
-    nonisolated public func getEndpoint(chain: Chain) throws -> String {
+    nonisolated public func getEndpoint(chain: String) throws -> String {
```

Comment:
```suggestion
    nonisolated public func getEndpoint(chain: String) throws -> String {
```

let's just stick to `Chain` for now